src/__experimental__/components/checkbox/checkbox-classic.style.js

```diff
@@ -0,0 +1,90 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+import StyledCheckableInput from '../checkable-input/checkable-input.style';
+import FieldHelpStyle from '../field-help/field-help.style';
+import HiddenCheckableInputStyle from '../checkable-input/hidden-checkable-input.style';
+import LabelStyle from '../label/label.style';
+import StyledHelp from '../help/help.style';
+
+export default ({
+  checked, disabled, fieldHelpInline, reverse, theme
+}) => theme.name === THEMES.classic && css`
+  ${StyledCheckableInput} {
+    padding: 1px 0 0 0;
+  }
+
+  ${HiddenCheckableInputStyle},
+  svg {
+    padding: 1px;
+  }
+
+  ${StyledCheckableInput},
+  ${HiddenCheckableInputStyle},
+  ${HiddenCheckableInputStyle} + svg {
+    height: 15px;
+    width: 15px;
+  }
+
+  ${FieldHelpStyle} {
+    margin-left: 15px;
+  }
+
+  ${LabelStyle} .carbon-icon::before {
+    content: '\\E943';
```

Comment:
Shouldn't that icon be rendered with the `Icon` component?
I think that all icons should be defined in one place and then reused.

Reply:
I did it this way as doing it properly would require making functional changes to the component purely for implementing the classic theme.

As everybody keeps saying the Classic theme is only temporary and should be kept as separate as possible, this felt like the best way to implement it without having to make changes elsewhere.